packages/messaging/src/controllers/controller-interface.ts

```diff
@@ -61,7 +63,7 @@ export default class ControllerInterface {
    * @return {Promise<string> | Promise<null>} Returns a promise that
    * resolves to an FCM token.
    */
-  getToken() {
+  async getToken() {
```

Comment:
@jshcrowthe does Firebase polyfill async and await?

Reply:
Since this is in a typescript file, the build process compiles down to ES5 which will result in the inclusion of the regenerator runtime.

Firebase itself doesn't have to polyfill these, but your runtime cost will go up.

Reply:
so i'm guessing the suggested way going forward is not to use async/await in the the source, but tests are ok?

Reply:
yes @pinarx that's right